packages/vscode-extension-pmml-editor/src/extension/extension.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {backendI18nDefaults, backendI18nDictionaries} from "@kogito-tooling/backend/dist/i18n";
+import {VsCodeBackendProxy} from "@kogito-tooling/backend/dist/vscode";
+import {I18n} from "@kogito-tooling/i18n/dist/core";
+import * as KogitoVsCode from "@kogito-tooling/vscode-extension";
+import * as vscode from "vscode";
+
+let backendProxy: VsCodeBackendProxy;
+
+export function activate(context: vscode.ExtensionContext) {
+    console.info("Extension is alive.");
+
+    const backendI18n = new I18n(backendI18nDefaults, backendI18nDictionaries, vscode.env.language);
+    backendProxy = new VsCodeBackendProxy(context, backendI18n);
+
+    KogitoVsCode.startExtension({
+        extensionName: "kie-group.vscode-extension-pmml-editor",
```

Comment:
Since I am seeing 4 spaces per indentation-level (other files have 2), 
I believe this file is not being formatted with `prettier`.

Reply:
Oops.. fixed. `prettier` now configured in my IDE.